compiler/env/TRPersistentMemory.cpp

```diff
@@ -73,3 +74,26 @@ TR_PersistentMemory::TR_PersistentMemory(
    _totalPersistentAllocations()
    {
    }
+
+void
+TR_PersistentMemory::printMemStats()
+   {
+   fprintf(stderr, "TR_PersistentMemory Stats:\n");
+   for (uint32_t i = 0; i < TR_MemoryBase::NumObjectTypes; i++)
+      {
+      fprintf(stderr, "\t_totalPersistentAllocations[%s]=%d\n", objectName[i], _totalPersistentAllocations[i]);
```

Comment:
`_totalPersistentAllocations[i]` is defined as `size_t` so  `%d` is not quite correct.
Ideally one should use `%zu`, but last time I tried (a few years back) one of the platforms didn't support it. The best way is probably to use `%lu` and cast it to `unsigned long`